scripts/setup-tools-for-debian.sh

```diff
@@ -36,7 +39,9 @@ helm init -c --stable-repo-url https://charts.helm.sh/stable
 sudo apt-get install -y socat
 
 # kubectl
-curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+# Keep it in sync with the version declared in scripts/setup-minikube-for-linux.sh
+KUBERNETES_VERSION="v1.18.18"
```

Comment:
Could you pull this directly from `scripts/setup-minikube-for-linux.sh` itself, just to avoid there being a mismatch?

Reply:
For other reasons (mainly the helm version used), it is better not to use the same version. I will update helm in another PR and try to keep the versions in a single place as you suggested. :-)